### DISCUSSION TITLE
Feature Request: Fuzzy Search of Variables in Dashboard

### DISCUSSION THREAD

**Discussion by kk6mrp** [grafana/grafana#55007] [Dashboards, Panels & Visualizations]
I have a large list of variables that are getting read from my database and it would be superb to have a fuzzy search on this search box! As it is, the user must be much more familiar with the exact name to narrow down the search than if it were fuzzy.

**Comment by paulwer** [comment]
we are looking for such feature too.
would be great :)

**Reply by bob454522** [reply]
Ive been wanting this for a while,  also Case-Insensitive search should be the norm across ALL of grafana (specifically when using the Query panel against a Graphite data source - its pretty frustrating that case interferes with the auto fill there).  
thanks,
LOVE GRAFANA!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
